client_grpc.go

```diff
@@ -120,9 +135,12 @@ func (c *clientGrpc) NewQueryClient() (enginev1connect.QueryServiceClient, error
 		)
 		endpoint = c.config.apiServer.Value
 	}
-
+	client := c.httpClient
+	if strings.HasPrefix(endpoint, "http://127.0.0.1") || strings.HasPrefix(endpoint, "http://localhost") {
```

Comment:
@rooftoofwoof can you make it look for `http://` instead?

Reply:
I would like to stop having differentiated behavior based on "localhost" or "127.0.0.1", but "http://" is principled.